microsoft-edge/devtools-guide-chromium/whats-new/2021/07/devtools.md

```diff
@@ -0,0 +1,288 @@
+
+Previously, users of screen readers would only hear the number of errors, warnings, or issues announced in the DevTools toolbar.  The additional information of what type of notification was being announced wasn't included, such as "Error, Warning, or Issue". For example, if the DevTools reported 3 errors, screen readers would just announce "3".
+
+Now in Microsoft Edge version 93, screen readers correctly announce the number of errors, warnings, or issues that the DevTools have detected and the type of notification. For example, if the DevTools reported 3 errors, screen readers now announce "3 errors".  This fix has been applied to both the notifications in the DevTools toolbar and in the Console.
+
+:::image type="complex" source="../../media/2021/07/screen-reader-errors-warnings-issues.msft.png" alt-text="The errors, warnings, and issues UI in the toolbar and Console" lightbox="../../media/2021/07/screen-reader-errors-warnings-issues.msft.png":::
+   The errors, warnings, and issues UI in the toolbar and **Console**
+:::image-end:::
+
+<!-- It'd be good to have a video of this a11y fix where the text that the screen reader announces is displayed -->
+
+For information about debugging Console errors, navigate to [Debug errors reported in Console](../../../console/console-debug-javascript.md).  For information about issues found by DevTools and improvements you can make to a webpage, navigate to [Find and fix problems using the Issues tool](../../../issues/index.md).
+
+To review the history of this feature in the Chromium open-source project, navigate to Issue [1223208][CR1223208].
+
+
+<!-- 9 -->
+## Copy as PowerShell in the Network tool now includes cookies
+
+<!-- Title: Generate PowerShell commands for network requests in the Network tool -->
+<!-- Subtitle: The Copy as PowerShell context menu option now correctly sets the user-agent string and cookies when generating PowerShell network requests. -->
+
+Previously, in the **Network** tool, the **Copy** > **Copy as PowerShell** context menu option didn't include cookies when generating a PowerShell command for a given network request in the Network Activity Log. This meant that the generated PowerShell command couldn't successfully make the same network request if cookies were required.
+
+Now in Microsoft Edge version 93, the **Copy as PowerShell** context menu option correctly sets the User-Agent string and cookies that were observed by DevTools.  The generated PowerShell command can now successfully make the same network request that was observed by DevTools, even to servers that depend on cookies.
+
+:::image type="complex" source="../../media/2021/07/copy-as-powershell.msft.png" alt-text="The Copy as PowerShell command" lightbox="../../media/2021/07/copy-as-powershell.msft.png":::
+   The **Copy as PowerShell** command
+:::image-end:::
+
+For more information about the Network Activity Log, navigate to [Network Analysis reference](../../../network/reference.md).
```

Comment:
Same comment as above. Can we merge this sentence and the next one into one paragraph?